packages/adapter-neon/README.md

```diff
@@ -1,29 +1,14 @@
-# @prisma/adapter-neon
+# Prisma driver adapter for Neon serverless driver
 
-Prisma driver adapter for [Neon Serverless Driver](https://github.com/neondatabase/serverless).
+Prisma driver adapter for [Neon Serverless Driver](https://github.com/neondatabase/serverless). Refer to the [announcement blog post](https://www.prisma.io/blog/serverless-database-drivers-KML1ehXORxZV) and our [docs](https://www.prisma.io/docs/guides/database/neon#how-to-use-neons-serverless-driver-with-prisma-preview) for more details.
 
-See https://github.com/prisma/prisma/releases/tag/5.4.0 and https://www.prisma.io/blog/serverless-database-drivers-KML1ehXORxZV for details.
+> **Note:**: Support for Neon's serverless driver is available from Prisma versions [5.4.2](https://github.com/prisma/prisma/releases/tag/5.4.0) and later.
 
-The following usage tutorial is valid for Prisma 5.4.2 and later versions.
+The [Neon serverless driver](https://github.com/neondatabase/serverless) is a low-latency Postgres driver for JavaScript and TypeScript that allows you to query data from serverless and edge environments over HTTP or WebSockets.
```

Comment:
I'm not sure I would mention `HTTP` here, considering how little support Prisma can provide for it

Reply:
I can omit both HTTP and WebSockets in the meantime 🙂